src/js/components/TaskTable.js

```diff
@@ -165,6 +184,12 @@ class TaskTable extends React.Component {
     );
   }
 
+  renderHost(prop, task) {
+    let marathonTask = MarathonStore.getTaskFromTaskID(task.id);
+
+    return <TaskEndpointsList portLimit={3} task={marathonTask} />;
```

Comment:
can we add a `key` here with the task id or something? i'm not 100% clear on this, but if we add a unique key, then react will know not to re-create the component or something. but let's just do it because we have nothing to lose